server/fixtures/models/article.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  title: 'Hello world!',
+  body: 'Just a dummy data',
```

Comment:
Thee are going to be multiple objects in this file. So this should not be a default export